akka-docs/rst/scala/code/docs/akka/typed/IntroSpec.scala

```diff
@@ -121,15 +121,14 @@ class IntroSpec extends TypedSpec {
         chatRoom ! GetSession("ol’ Gabbler", gabblerRef)
 
         Stateful(
-          behavior = (_, _) ⇒ Unhandled,
-          signal = { (ctx, sig) ⇒
+          onMessage = (_, _) ⇒ Unhandled,
```

Comment:
good rename, I hope we'll revisit the possibility of single arg ones at some point still :)